examples/old/maxcut_sdp.jl

```diff
@@ -18,21 +18,20 @@
 #############################################################################
```

Comment:
Why is this file still in the `old/` directory?

Reply:
It produces errors regarding Cholesky decomposition.

Reply:
See https://github.com/JuliaOpt/JuMP.jl/issues/1547